### ISSUE TITLE
a PROBLEM when using VGG as motion model

### ISSUE THREAD

**Issue by jessico** [bryanyzhu/two-stream-pytorch#53]
Hi,I try to use VGG16  as motion-model to fine-tune on ucf101 dataset,first stack 10 x-axis optical folw pics and 10 y-axis pics to get a 20 pics clip,then send it to VGG16 pretrained on ImagNet(change the in_channel from 3 to 20 and change the last classifier layer to 101)to do classification task. I met the same question as https://github.com/bryanyzhu/two-stream-pytorch/issues/1. 
But when I use ResNet architechiture,everything goes well,does it correspoding to the preprocess on optical-flow pics?I dont normalize the optiflow pics.
my training acc1 and test acc1 is around 1% and dosnt go up.
